apps/public/views.py

```
@@ -1,11 +1,8 @@
 # apps/public/views.py
 
 from django.http import HttpResponse
-from django.shortcuts import render
-
-from apps.certification_bodies.models import CertBody  # Correct import
-from apps.organizations.models import Certification  # Remove unused import
-
+from django.shortcuts import render, get_object_or_404
```

Ruff (0.8.2)

`django.shortcuts.get_object_or_404` imported but unused

Remove unused import: `django.shortcuts.get_object_or_404`

(F401)